models/stg_hubspot__deal.yml

```
@@ -20,8 +20,8 @@ models:
         description: Used to determine the order in which the stages appear when viewed in HubSpot.
       - name: is_active
         description: Whether the pipeline stage is currently in use.
-      - name: is_closed_won
-        description: Whether the stage represents a Closed Won deal.
+      - name: is_closed
+        description: Boolean indicating if the pipeline stage is closed. For the state of the closed deal, refer to the pipeline_stage_label column.
```

```suggestion
        description: Boolean indicating if the pipeline stage is closed. For the state of the closed deal ("Closed Won" vs "Closed Lost", refer to the pipeline_stage_label column.
```
Could be nice to clarify